pages/company/visualizations/gh_company_affiliation.py

```diff
@@ -0,0 +1,275 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import color_seq
+from queries.company_query import company_query as cq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+from fuzzywuzzy import fuzz
+
+PAGE = "company"  # EDIT FOR PAGE USED
+VIZ_ID = "gh-company-affiliation"  # UNIQUE IDENTIFIER FOR CALLBAKCS, MUST BE UNIQUE
+
+paramter_1 = "company-contributions-required"
+paramter_2 = "null-check"
+
+
+gc_gh_company_affiliation = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Company Affiliation by Github Account Info",
```

Comment:
Right now we're only looking at github but should this be made more general? e.g. "User Account Company Affiliation"

Reply:
The reason I feel like it is important to specify is bc I think there is users who might infer that we are doing other things on the backside to get the information besides specifically in the github account. Definitely plan on listing this one last tho bc it gives the least amount of info imo

Reply:
That makes sense to me. maybe "User-Volunteered Affiliation" or something? and then we say what the source is in the info thing? Definitely can stay as it is, just playing with the language to find the right fit.